vnext/PropertySheets/External/Microsoft.ReactNative.Uwp.CSharpModule.props

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- 
+  Copyright (c) Microsoft Corporation. All rights reserved.
+  Licensed under the MIT License.
+-->
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <ReactNativeWindowsDir Condition="'$(ReactNativeWindowsDir)' == ''">$([MSBuild]::NormalizePath('$(MSBuildThisFileDirectory)\..\..'))\</ReactNativeWindowsDir>
```

Comment:
can you clarify where this props goes? is this going into node-modules? isn't this missing react-native-windows at the end?

Reply:
I plan on adding new module templates to the CLI, and just like we just added targets/props for new cli apps to consume, these props/targets are for new cli modules. I also plan on updating existing community modules to load these targets, which I've done locally for react-native-camera.

Reply:
Someone added Libs targets, switching to them.